hs_core/models.py

```diff
@@ -2103,7 +2103,7 @@ def sysmeta_path(cls, resource_id):
         """Get URL of resource map xml."""
         return "{resource_id}/data/resourcemap.xml".format(resource_id=resource_id)
 
-    def delete(self, using=None):
+    def delete(self, using=None, keep_parents=False):
```

Comment:
what is keep_parents?  It doesn't look like it's even used.

Reply:
This is the  Django model `delete()` method (which has the keep_parents argument) we are overriding here. Neither the 'using' nor the keep_parents' arguments are being used as they have the default values. However, I think one could pass both these arguments when calling 'delete()' on super.